packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -187,10 +187,8 @@ export async function startTransformByQ() {
     } catch (error: any) {
         await transformationJobErrorHandler(error)
     } finally {
-        if (transformByQState.isCancelled()) {
-            await postTransformationJob()
-            await cleanupTransformationJob()
-        }
+        await postTransformationJob()
+        await cleanupTransformationJob()
```

Comment:
These should always be running since they reset the Transformation Hub and other things to prepare for a subsequent transformation run.

Reply:
finalizeTransformByQ (down the chain of humanInTheLoopRetryLogic for happy path) will make duplicated call to postTransformationJob and cleanupTransformationJob -> can this cause extra messages to be published to chat or extra telemetry to be emitted? 

```
export async function finalizeTransformByQ(status: string) {
    try {
        // Set the result state variables for our store and the UI
        await finalizeTransformationJob(status)
    } catch (error: any) {
        await transformationJobErrorHandler(error)
    } finally {
        await postTransformationJob()
        await cleanupTransformationJob()
    }
}
```
I think we can remove the postTransformationJob() and cleanupTransformationJob() from finalizeTransformByQ